app/controllers/home_controller.rb

```diff
@@ -50,9 +50,9 @@ def first_login_help
 
   # home page itself
   def index
-    @user_count = User.count
-    @work_count = Work.posted.count
-    @fandom_count = Fandom.canonical.count
+    @user_count = Rails.cache.fetch('User_count', expires_in: ArchiveConfig.HOMEPAGE_CACHE_TIME, race_condition_ttl: 5) { User.count }
```

Comment:
We generally use all lowercase in the cache fragment names, like user_count, work_count, fandom_count.